operate/schema/src/main/java/io/camunda/operate/schema/IndexSchemaValidator.java

```diff
@@ -166,6 +159,11 @@ public Map<IndexDescriptor, Set<IndexMappingProperty>> validateIndexMappings() {
     return newFields;
   }
 
+  private String getOperateIndexPattern() {
+    final String prefix = schemaManager.getIndexPrefix();
+    return (prefix != null && !prefix.isBlank() ? prefix + "-" : "") + OPERATE + "*";
```

Comment:
"".equals(prefix) might be easier 🤔

Reply:
yes, unless it contains whitespaces or is null :D I just took what we already use from [AbstractIndexDescriptor.java#L57](https://github.com/camunda/camunda/blob/main/webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/AbstractIndexDescriptor.java#L57) 

I will actually expose and reuse that function.